packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -674,7 +674,7 @@ class PlatformViewLayer extends Layer {
     }
     if (MouseTrackerAnnotation == S) {
```

Comment:
not for this PR but why isn't this `S is MouseTrackerAnimation`? @digiter

Reply:
`S` is a `Type` object and will never be a `MouseTrackerAnimation`. I don't think there's a way to test that `S` extends/implements `MouseTrackerAnimation`

Reply:
we should probably have it say `S == MouseTrackerAnnotation` to make it clearer that the variable here is `S`.

Reply:
Done